typescript/infra/scripts/warp-routes/monitor-warp-routes-balances.ts

```diff
@@ -0,0 +1,131 @@
+import { Connection } from '@solana/web3.js';
+import { ethers } from 'ethers';
+import { Gauge, Registry } from 'prom-client';
+
+import {
+  ERC20__factory,
+  SealevelHypCollateralAdapter,
+} from '@hyperlane-xyz/hyperlane-token';
+import { ChainMap, MultiProvider } from '@hyperlane-xyz/sdk';
+import { debug, objMap, promiseObjAll } from '@hyperlane-xyz/utils';
+
+import { startMetricsServer } from '../../src/utils/metrics';
+
+import { WarpTokenConfig, tokenList } from './token_config';
+
+const metricsRegister = new Registry();
+const warpRouteTokenBalance = new Gauge({
+  name: 'hyperlane_warp_route_token_balance',
+  help: 'HypERC20 token balance of a Warp Route',
+  registers: [metricsRegister],
+  labelNames: [
+    'chain_name',
+    'token_address',
+    'token_name',
+    'wallet_address',
+    'token_type',
+  ],
+});
+
+async function main(): Promise<boolean> {
+  startMetricsServer(metricsRegister);
+
+  const checkFreqeuncy = 1000; // 1 second
```

Comment:
parameterize this, and default to something more like every 10-20 seconds

Reply:
sorry i should have been more clear, when I meant parameterize, i meant a command line argument to the script or an ENV var so that it can be set without a code change